test/xpu_api/utilities/meta/meta.unary.prop.query/alignment_of.pass.cpp

```diff
@@ -0,0 +1,80 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/type_traits>
+
+#include "support/test_macros.h"
+#include "support/utils.h"
+
+#include <cstdint>
+
+#if TEST_DPCPP_BACKEND_PRESENT
+template <class T, unsigned A>
+void
+test_alignment_of(sycl::queue& deviceQueue)
+{
+    deviceQueue.submit([&](sycl::handler& cgh) {
+        cgh.single_task<T>([=]() {
+            const unsigned AlignofResult = TEST_ALIGNOF(T);
```

Comment:
As far as I see, `TEST_ALIGNOF` is an obsolete functionality that was needed when C++03 was supported. Since C++11, `TEST_ALIGNOF` is always equivalent to `alignof` keyword. 
I guess, `TEST_ALIGNOF` should be removed, of course as part of separate PR.
For this test, I propose to simply use `alignof` instead.